styles.css

```diff
@@ -0,0 +1,89 @@
+body{
+    text-align: center;
+    color: rgb(255, 255, 255);
+    background-position: center;
+ background-image: linear-gradient( 115deg, rgba(58, 58, 158, 0.8), rgba(136, 136, 206, 0.7) ), url(https://cdn.freecodecamp.org/testable-projects-fcc/images/survey-form-background.jpeg);
+    background-size: cover;
+    font-family: Arial, Helvetica, sans-serif;
+    font-weight: 800;
+    line-height: 2;
+    
+}
+
+form { 
+    max-width: 1000px;
+    font-size: 20px;
+    padding: 7px;
+    border: 2px solid rgb(255, 255, 255);
+    border-radius: 30px;
+    margin: auto;
+
+    
+}
+
+input{
```

Comment:
Please, I think It would be better to add linters to make your clean code! Please do not leave white spaces.